pkg/validation/options.go

```diff
@@ -303,10 +304,10 @@ func parseProviderInfo(o *options.Options, msgs []string) []string {
 		case o.JWTKey == "" && o.JWTKeyFile == "":
 			msgs = append(msgs, "login.gov provider requires a private key for signing JWTs")
 		case o.JWTKey != "":
-			// The JWT Key is in the commandline argument
+			// The JWT SessionKey is in the commandline argument
 			signKey, err := jwt.ParseRSAPrivateKeyFromPEM([]byte(o.JWTKey))
 			if err != nil {
-				msgs = append(msgs, "could not parse RSA Private Key PEM")
+				msgs = append(msgs, "could not parse RSA Private SessionKey PEM")
```

Comment:
I don't really understand this change? Was this intentional?

Reply:
No, I think that was Goland trying to be smart and failing 🤔